### DISCUSSION TITLE
AgentGroupChat with 'human' interaction support

### DISCUSSION THREAD

**Discussion by guru98** [microsoft/semantic-kernel#6027] [Ideas]
Hi, if I understand well autogen and semantic kernel agents are designed thinking to multi agent cooperation as a way to improve answer quality, segment complexity through agent sepcialization. Now in real world is clear that often you need human interaction to complete request informations but also to allow to take decisions about path and choices. My question is if you are thinking to a version of AgentGroupChat that allow human interaction. A sort of extentions of selection functions that merge flow orchestration.

**Comment by sophialagerkranspandey** [comment]
Yes, this is something that we are planning to do in the future. Looping in @crickman to elaborate further. thanks

**Comment by crickman** [comment]
Yes, I a _user-proxy_ agent is absolutely on my radar.  Its been proposed as part of the overall framework but didn't make the current cut-line.  As we explore patterns further, I very much aim to have this addressed.

In the current version, the termination criteria for the chat might include when user input is sought.  Once provided, the chat may be continued.

I'd really appreciate continuing to engage on this for your perspective and insight.

**Reply by unsafePtr** [reply]
I see it's closed, but was it implemented or is there any available example?

**Reply by listentorick** [reply]
Did anything come of this?
